tests/foreman/cli/test_discoveryrule.py

```diff
@@ -592,8 +437,9 @@ def class_user_manager(self, class_user_password, class_org, class_location):
         yield user
         try:
             user.delete()
-        except HTTPError:
-            logger.exception('Exception while deleting class scope user entity in teardown')
+        except HTTPError as err:
+            logger.exception(err)
+            logger.error('Exception while deleting class scope user entity in teardown')
```

Comment:
Why this block is required even? we don't have try except for other tests in robottelo for user deletion
```suggestion

```

Reply:
Yeah agree, but what's the issue if we have it here :)

Reply:
just unnecessary lines :)